master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calicoctl.yaml

```diff
@@ -13,10 +13,76 @@ metadata:
   namespace: kube-system
 spec:
   hostNetwork: true
+  serviceAccountName: calicoctl
   containers:
   - name: calicoctl
     image: quay.io/calico/ctl:{{site.data.versions[page.version].first.components["calicoctl"].version}}
     command: ["/bin/sh", "-c", "while true; do sleep 3600; done"]
     env:
     - name: DATASTORE_TYPE
       value: kubernetes
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: calicoctl
+rules:
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - pods
+      - nodes
+    verbs:
+      - get
+      - list
+      - update
+  - apiGroups: [""]
+    resources:
+      - pods/status
+    verbs:
+      - update
+  - apiGroups: ["extensions"]
+    resources:
+      - networkpolicies
+    verbs:
+      - create
+      - get
+      - list
+      - update
+      - delete
+  - apiGroups: ["crd.projectcalico.org"]
+    resources:
+      - bgppeers
+      - bgpconfigurations
+      - clusterinformations
+      - felixconfigurations
+      - globalnetworkpolicies
+      - hostendpoints
+      - ippools
+      - networkpolicies
+      - profiles
+      - workloadendpoints
+    verbs:
+      - create
+      - get
+      - list
+      - update
```

Comment:
For the CRDs I wonder if we also need `patch`, `deletecollection` and `watch`? Most likely not `watch`, but not sure if we use `patch` under the coder when we make a `PUT` call to the REST API? CC @doublek 
Maybe something we might need when we have the `calicoclt edit` command working? @robbrockbank @mgleung

Reply:
@gunjan5 we don't need any of those (yet).

We should be using patch, but we aren't (yet).